### PR TITLE
Rename sharing to streaming

### PR DESCRIPTION
**Pull request by BroderPeters**
Inspired by https://github.com/screego/server/commit/25aafefacc9cca016cc88bc81e1076ab5834c1e5#r43108176

### CODE REVIEW

File: ui/src/message.ts [hunk 1]
```diff
@@ -57,7 +57,7 @@ export interface RoomInfo {
 export interface RoomUser {
     id: string;
     name: string;
-    sharing: boolean;
+    streaming: boolean;
```
Comment (jmattheis) [inline ui/src/message.ts:60]: This interface only defines the payload created by the backend. This flag also needs to be adjusted in:

https://github.com/screego/server/blob/53a647e15881abf1099e9bc67b05a7764c5fe667/ws/outgoing/messages.go#L19-L25

**Review by jmattheis — CHANGES_REQUESTED** [review]
Hey! Thanks for your contribution. 

I've added a remark in a review comment.

File: ui/src/message.ts (hunk 1 above)
Comment (BroderPeters) [reply ui/src/message.ts:60]: Will adjust and this time also build and test!

**Comment by BroderPeters** [comment]
@jmattheis I fixed the necessary points in code in order to successfully build and execute.

There are still some places left like `startSharing` or `endShare` (which also seem inconsistent in terms of the end_ing_ *rogl), but I'm not really sure whether it makes sense to also adjust them as they describe more of the action itself.

**Comment by jmattheis** [comment]
> @jmattheis I fixed the necessary points in code in order to successfully build and execute.
> 
> There are still some places left like `startSharing` or `endShare` (which also seem inconsistent in terms of the end_ing_ *rogl), but I'm not really sure whether it makes sense to also adjust them as they describe more of the action itself.

I think these events should be adjusted as well. But this can be done separately.

**Review by jmattheis — APPROVED** [review]
Thanks!

**Pull request merged**